### ISSUE TITLE
Accuracy not improving for MNIST example with MXnet backend

### ISSUE THREAD

**Issue by jackgle** [awslabs/keras-apache-mxnet#242]
I have an AWS EC2 c4.2xlarge instance with Deep Learning Ubuntu AMI. When using the built-in environment mxnet_p36 and running the MNIST example here (https://github.com/awslabs/keras-apache-mxnet/blob/master/examples/mnist_cnn.py), I see no improvement in accuracy beyond 10% after several epochs. Switching to the tensorflow_p36 environment and running the same script obtains 70%+ over an epoch. I also tried installing mxnet-mkl and upgrading keras-mxnet. Please let me know of any ideas for this issue.

**Comment by roywei** [comment]
could you try upgrading to latest MXNet 1.5.0 and Keras-MXNet 2.2.4.2? It should be working. Thanks
